index.html

```diff
@@ -2484,6 +2484,84 @@ <h3>serviceEndpoint</h3>
                   </pre>
       </section>
 
+      <section id="x509Certificate" about="https://w3id.org/security#x509Certificate"
+      typeof="rdf:Property">
+        <h3>x509Certificate</h3>
+        <p>
+          The x509Certificate property is used to associate an <a href="https://tools.ietf.org/html/rfc5280">X.509 Certificate</a> with a proof.
```

Comment:
This property should specify the encoding format for the value. I suggest we use multibase.

Reply:
Agreed, however because multi-base provides a surface for many choices of encoding we should consider providing implementation guidance around usage of the property in the linked data proof specification (e.g please consider using x encoding via multi-base)

Reply:
> we should consider providing implementation guidance around usage of the property in the linked data proof specification

Yes, agreed... we may even want to do that in the vocabulary? I expect us to suggest base64pad (which I believe is the default for X509 certs?)... other options to limit to might be base64url or base58?

Reply:
For the certificate I was thinking we could define it as base64url then the PEM conversion is trivial?

Reply:
For instance the certificate given in the example is a just the base64url contents from a PEM encoded cert

Reply:
> For the certificate I was thinking we could define it as base64url then the PEM conversion is trivial?

Well, PEM uses base64pad, not base64url...

I believe the current authoritative spec for certificate encoding is [RFC7468](https://tools.ietf.org/html/rfc7468)... which doesn't use base64url... it uses base64pad... with headers and trailers. See https://tools.ietf.org/html/rfc7468#section-2 for more information. 

I'm still torn about breaking rank with PEM encoding... which does specify `----- BEGIN` and `----- END` headers and trailers. If we do that, we can't compress down to binary for CBOR-LD use cases... but then again, we probably won't be compressing `x509Certificate` values in CBOR-LD. The same is not true for `x509CertificateFingerprint`.

So, we could either just follow the rules in https://tools.ietf.org/html/rfc7515#section-4.1.6 -- you'll want to make this a `@list` in the JSON-LD Context as order matters for `x509Certificate`. The upside here is it just does what JOSE does... the downside is that we won't be able to CBOR-LD compress the value (not that doing so is a regular use case?).

The other option is multibase-encode the PEM certificate, still use `@list` to be able to encode certificate chains, and don't support PEM header/trailers.

@dlongley @tplooker, thoughts?

Reply:
> The other option is multibase-encode the PEM certificate, still use @list to be able to encode certificate chains, and don't support PEM header/trailers.

There is already precedence via https://tools.ietf.org/html/rfc7515#section-4.1.6 for doing base64(pad)-encoded DER (i.e., stripping the PEM armor). So, at minimum we should be able to do an `@list` with values that have that encoding. However, I think we should take it a step further to enable compression in CBOR-LD by making each value multibase encoded (one extra character prefixed before each base64pad value).